runtime/compiler/aarch64/codegen/J9CodeGenerator.cpp

```diff
@@ -48,6 +48,8 @@ J9::ARM64::CodeGenerator::CodeGenerator() :
       TEMPORARY_initJ9ARM64TreeEvaluatorTable(cg);
       initTreeEvaluatorTable = true;
       }
+
+   self()->setSupportsDivCheck();
```

Comment:
Sorry--I just noticed after I approved...   `cg` is already used in this constructor.  Can you replace your `self()` with `cg` for consistency?

Reply:
Changed to `cg`.